community/people/aps/index.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: person
+title: Andrew Schumaker
+role: Graduate Student
+image: Andrew_Github.jpg
+email: aschumaker@wisc.edu
+phone: "218/4608408"
```

Comment:
Also including personal phone (I assume) is probably not necessary, but up to personal preference.